pom.xml

```diff
@@ -836,16 +903,10 @@
                 <version>3.6.2.Final</version>
             </dependency>
 
-            <!-- manage some hadoop dependencies -->
-            <dependency>
-                <groupId>commons-beanutils</groupId>
-                <artifactId>commons-beanutils</artifactId>
-                <version>1.9.4</version>
-            </dependency>
```

Comment:
i would keep this here, with the comment, instead of moving it up (unless there's a reason to move it up)

Reply:
Can do - moved it back down with the comment.